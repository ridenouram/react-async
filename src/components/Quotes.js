import React from 'react';
import PropTypes from 'prop-types';
import Quote from './Quote';

export default function Quotes({ quotes }) {
  const quoteCards = quotes.map(quote => (
    <Quote key={quote.quote} character={quote.character} image={quote.image} quote={quote.quote} />
  ));

  return (
    quoteCards 
  );
}

Quotes.propTypes = {
  quotes: PropTypes.array.isRequired
};
