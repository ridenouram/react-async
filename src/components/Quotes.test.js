import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quote component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Quotes quotes={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
