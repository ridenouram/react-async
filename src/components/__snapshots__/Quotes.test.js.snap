// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component matches snapshot 1`] = `null`;
